pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -211,7 +212,7 @@ type endpointController struct {
 }
 
 func newController(ctx context.Context, url string, batchPeriod time.Duration) *endpointController {
-	client := clientset.NewForConfigOrDie(&restclient.Config{Host: url, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}})
+	client := clientset.NewForConfigOrDie(&restclient.Config{Host: url, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}, ContentType: runtime.ContentTypeJSON}})
```

Comment:
why did this have to hard-code json?

Reply:
Test logic makes assertions on requests and assumes they are JSON.